src/App/Property.cpp

```diff
@@ -62,6 +62,8 @@ Property::~Property()
 
 const char* Property::getName(void) const
 {
+    if (myName == NULL)

```

Comment:
The C++ Core Guidelines suggest this would be better written:
```suggestion
    if (!myName)
```
(see http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es87-dont-add-redundant--or--to-conditions)